test/test_helper.rb

```diff
@@ -0,0 +1,69 @@
+# For get coverage local - uncomment 4-7 lines
+require 'coveralls'
+Coveralls.wear!('rails')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.